app/src/main/java/com/keylesspalace/tusky/components/account/AccountActivity.kt

```diff
@@ -854,6 +870,30 @@ class AccountActivity : BottomSheetActivity(), ActionButtonActivity, HasAndroidI
                     )
                 }
             }
+            R.id.action_share_account_link -> {
+                // If the account isn't loaded yet, eat the input.
+                if (loadedAccount != null) {
+                    val url = loadedAccount!!.url
+                    val sendIntent = Intent()
+                    sendIntent.action = Intent.ACTION_SEND
+                    sendIntent.putExtra(Intent.EXTRA_TEXT, url)
+                    sendIntent.type = "text/plain"
+                    startActivity(Intent.createChooser(sendIntent, resources.getText(R.string.send_account_link_to)))
+                }
+                return true
+            }
+            R.id.action_share_account_username -> {
+                // If the account isn't loaded yet, eat the input.
+                if (loadedAccount != null) {
+                    val fullUsername = loadedAccount!!.fullUsername
```

Comment:
instead of the `!= null` and then `!!` pattern you could 
```
loadedAccount?.let { account ->
// do something with the non-null account
}
```

Reply:
The way I wrote it matches adjacent menu items such as `action_open_in_web`. So I'd prefer to either keep it the way it is or refactor the entire `when` block to use `.let` every time.

Reply:
Its not really important, however you like